schema/table.go

```diff
@@ -248,55 +281,50 @@ func (t *Table) ValidateName() error {
 	return nil
 }
 
-// GetAddedColumns returns a list of columns that are in this table but not in the other table.
-func (t *Table) GetAddedColumns(other *Table) []Column {
-	var added []Column
-	for _, c := range t.Columns {
-		if other.Columns.Get(c.Name) == nil {
-			added = append(added, c)
-		}
-	}
-	return added
-}
+// added columns with no constraints - migrate // table.GetAddedColumns(other)
```

Comment:
These comments seem a bit outdated, but I think:

```
Destrucitve changes are done only in force mode

added columns:
 - no constraints -> migrate
 - not null -> drop table
  - unique -> drop table

changed columns:
 - only type change -> drop/add column
 - added constraint -> drop table
 - removed constraint -> remove constraint (e.g. we remove a not null from a column that can be null). OK to drop a user constraint in forced mode (we can document it)

removed columns:
  - no constraints -> do nothing, unless forced, then drop
  - with constraints -> drop column, see https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/postgresql/client/migrate.go#L103
```

Reply:
`Destructive` 😉

Reply:
This is a non-relevant comment - removed. I was just taking some notes and forgot to remove it.